modules/formulize/class/data.php

```diff
@@ -983,7 +983,7 @@ function changeUserSubmittedValues($element_id_or_handle, $newValues) {
 				}
 				break;
 		}
-		$prefix = ($ele_type == "check" OR ($ele_type == "select" AND $ele_value[1])) ? "#*=:*" : ""; // multiple selection possible? if so, setup prefix

```

Comment:
Actually, it's a little more complicated than this. One element type uses one delimiter and another element type uses the other delimiter. They should probably both use the same delimiter, but then a whole bunch of data in existing systems needs to be converted, and that might break something. It would also mean that the delimiter string through all the code needs to be changed. At this point, it's probably just easier to detect the element type here and use the appropriate delimiter string.